rotkehlchen/csv_exporter.py

```diff
@@ -508,3 +512,32 @@ def create_files(self, dirpath: Path) -> Tuple[bool, str]:
             return False, str(e)
 
         return True, ''
+
+    def create_zip(self) -> Optional[str]:
+        if not self.create_csv:
+            return None
+        dirpath = Path(mkdtemp())
```

Comment:
Add a comment and issue about managing to delete this once it's downloaded. Normally (as also seen in other places in the code) we would have used a context manager (a with statement) with this: https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

Reply:
Issue created #1853